xml/dm_registers.xml

```diff
@@ -966,13 +962,24 @@
     </register>
 
     <register name="Custom Features 0" short="custom0" address="0x70">
-        This optional register may be used for non-standard features. Future
-        version of the debug spec will not use this address.
-    </register>
-
-    <register name="Custom Features 15" short="custom15" address="0x7f">
-        This optional register may be used for non-standard features. Future
-        version of the debug spec will not use this address.
-    </register>
+        The optional {\tt custom} registers may be used for non-standard
+        features. Future version of the debug spec will not use these
+        addresses.
```

Comment:
To be consistent with the data0-11 and progbuf0-15 register blocks, should this say something like "The optional \RdmCustomZero through \RdmCustomFifteen may be used..."

Reply:
I agree.  Also, "Future versions of the debug spec" instead of "version".  Otherwise it looks great.

Reply:
Done. Thanks.